strings/names/wizardfirst.txt

```diff
@@ -1,37 +1,32 @@
 Alatar
+Anior
+Azor
 Archchancellor
 Boccob
 Circe
-Dumbledor
 Elminister
-Gandalf
 Grimm
 Gulstaff
-Houdini
+Ivras
+Kabahn
 Jim
 Kaschei
 Khelben
 Kreol
 Lina
-Merlin
 Mogan
-Mordenkainen
 Morgan
-Mystryl
-Nihilus
+Obeus
+Obras
 Palando
-Prospero
-Radagast
+Peleus
 Raistlin
-Rasputin
 Rincewind
```

Comment:
see what i said about the ninja. You've missed some, no need to remove them all anyways